src/tribler-core/tribler_core/components/implementation/tunnels.py

```diff
@@ -47,6 +49,7 @@ async def run(self):
             tunnel_cls = TriblerTunnelCommunity
 
         provider = DHTCommunityProvider(dht_discovery_community, config.ipv8.port) if dht_discovery_community else None
+        exitnode_cache = Path(config.state_dir) / "exitnode_cache.dat"
```

Comment:
Forcing `config.state_dir` to `Path` is unnecessary: `TriblerConfig` object ensures the thing is always a `Path`.